Добавляет перевод термина Drag-and-drop (#461)
web-standards-ru/dictionary: dictionary.md

**докта́йп,** конструкция вида `<DOCTYPE html>` в начале документа, которая обозначает используемую версию HTML. Используется браузерами для выбора режима ([стандартного](#standards-mode-standards-compliance-mode), [почти стандартного](#almost-standards-mode) или [совместимости](#quirks-mode)).

> ### drag-and-drop

Reviewer: ```suggestion
### drag and drop
```

> ### drag-and-drop, drag’n’drop
> 
> **драг-н-дроп**, способ взаимодействия с интерфейсом путём перетягивания элементов.

Reviewer 2: ```suggestion
### drag-and-drop

**драг-н-дроп, перетаскивание,** способ взаимодействия с интерфейсом, в котором пользователь перемещает объекты из одного места в другое в рамках страницы или снаружи, напр. _we can sort this list via drag-and-drop — мы можем отсортировать этот список с помощью драг-н-дропа_ или _to upload a file drag and drop it here — чтобы загрузить файл, перетащите его сюда._
```

> **драг-н-дроп, перетаскивание,** способ взаимодействия с интерфейсом, в котором пользователь перемещает объекты из одного места в другое в рамках страницы или снаружи, напр. _we can sort this list via drag-and-drop — мы можем отсортировать этот список с помощью драг-н-дропа_ или _to upload a file drag and drop it here — чтобы загрузить файл, перетащите его сюда._

Reviewer: ```suggestion
**драг-н-дроп, перетаскивание,** способ взаимодействия с интерфейсом, когда пользователь перемещает объекты из одного места в другое в рамках страницы или за пределами окна браузера, напр. _we can sort this list via drag-and-drop — мы можем отсортировать этот список с помощью драг-н-дропа_ или _to upload a file drag and drop it here — чтобы загрузить файл, перетащите его сюда._
```
